src/Autosuggest.vue

```diff
@@ -315,7 +315,14 @@ export default {
     updateCurrentIndex(index) {
       this.currentIndex = index;
     },
-    onDocumentMouseUp() {
+    onDocumentMouseUp(e) {
+      
+      /** Do not re-render list on input click */
+      const isChild = isDescendant(this.$el, e.target);
```

Comment:
`isDescendant` can be removed in favor of native `.contains` method. 

Please refactor to `this.$el.contains(e.target);`, remove `isDescendant` function from utils and I will approve!